src/main/java/hudson/plugins/xvnc/Xvnc.java

```diff
@@ -92,7 +95,9 @@ public void setUp(Context context, Run<?, ?> build, FilePath workspace, Launcher
 
         String cmd = Util.nullify(DESCRIPTOR.xvnc);
         if (cmd == null) {
-            cmd = "vncserver :$DISPLAY_NUMBER -localhost -nolisten tcp";
+            cmd = "vncserver :$DISPLAY_NUMBER -localhost -nolisten tcp " + (this.additionalArgs != null ? this.additionalArgs : "");
+        } else {
+            DESCRIPTOR.xvnc += " " + (this.additionalArgs != null ? this.additionalArgs : "");
```

Comment:
Fixed. However, I am not sure, what you mean with "sensitive to argument options".
Do you point at problems like assuring the correct order of arguments as the vncserver cli might not allow arbitrary ordering?
Regarding an additional macro to be able to assure a specific insertion point for the addtional parameters in relation to what already exists in the global configuration, I would propose to see it as a new feature.
I think the current feature already gives a lot of freedom and benefits to solve whatever you want. Surely, you have to get in contact with your vncserver installation and its cli parameter docs to use this feature such that you might even understand potential problems of argument ordering and so on. For me this will not cause any harm or less usability.